### ISSUE TITLE
Add Learning Resources for Developers 📚

### ISSUE THREAD

**Issue by jatin2003** [jatin2003/resources-for-web-developers#6]
What you think about adding some resources for developers to learn about web development like adding a section for Docs, platforms like freecodecamp, khanacademy, etc. - @ritvij14 & amankr1619

**Comment by ritvij14** [comment]
Good idea @jatin2003. Also, in the docs section, we could also add a part with names and GitHub links to various different and famous libraries/frameworks? I know it might be pretty big considering their number but it would be a good dictionary for people.

Also, I will add a few youtube channels which helped me out a lot if you don't mind.
